public/stylesheets/style.css

```diff
@@ -1,128 +1,216 @@
 body{
-  margin-bottom: 20rem;
+	margin-bottom: 20rem;
 }
 footer{
-  bottom : 0;
-  background-color : #FFF;
-  z-index : 1;
+	bottom : 0;
```

Comment:
Try to `undiff` whitespace changes like this if you didn't change the code.  Changing the whitespace can have consequences, some bracket-less languages really care about this and it is a bad habit to form.